### PR TITLE
Bug 1986410: Control CPUs amount under latency-test pod

### PR DESCRIPTION
**Pull request by Tal-or**
By having the new LATENCY_TEST_CPUS env variable, the user will be able to specify the amount of CPUs
that will be used by the pod which runs the latency tests.

The LATENCY_TEST_CPUS env variable is optional, the default value is the total amount of isolated CPUs minus one (this behavior is identical to the prior one).

No need to bump PAO under ```go.mod``` it's already updated.

Signed-off-by: Talor Itzhak <titzhak@redhat.com>

### CODE REVIEW

File: cnf-tests/pod-utils/cyclictest-runner/main.go [hunk 1]
```diff
@@ -29,10 +28,6 @@ func main() {
 		klog.Fatalf("failed to get self allowed CPUs: %v", err)
 	}
 
-	if *threads == -1 {
```
Comment (Tal-or) [inline cnf-tests/pod-utils/cyclictest-runner/main.go:32]: This is not needed anymore. The number of CPUs will be set by the user using the new variable and we'll always run the tool on every CPU which was allocated under the pod.

**Comment by cynepco3hahue** [comment]
Please run make install-commit-hooks and git commit --amend to update the title of the commit accordingly
It should include cnf-tests: ... IIRC
Also update dependencies to include your PAO changes

**Comment by Tal-or** [comment]
> Please run make install-commit-hooks and git commit --amend to update the title of the commit accordingly
> It should include cnf-tests: ... IIRC

done 

> Also update dependencies to include your PAO changes

Already done by someone else

**Comment by Tal-or** [comment]
/retest

```
 [test_id:34080] should be supported and available for the container usage [It]
    /go/src/github.com/openshift-kni/cnf-features-deploy/vendor/github.com/openshift-kni/performance-addon-operators/functests/1_performance/hugepages.go:84
    Unexpected error:
      
        <*strconv.NumError | 0xc001306060>: {
            Func: "Atoi",
            Num: "",
            Err: <*errors.errorString | 0xc0001201e0>{s: "invalid syntax"},
        }
        strconv.Atoi: parsing "": invalid syntax
    occurred
```
An empty string returned from the remote exec command. 
There is an open bug on that.

**Comment by Tal-or** [comment]
/retest
```
[rfe_id:27368][performance]
/go/src/github.com/openshift-kni/cnf-features-deploy/vendor/github.com/openshift-kni/performance-addon-operators/functests/1_performance/performance.go:57
  Tuned kernel parameters
  /go/src/github.com/openshift-kni/cnf-features-deploy/vendor/github.com/openshift-kni/performance-addon-operators/functests/1_performance/performance.go:286
    [test_id:28466][crit:high][vendor:cnf-qe@redhat.com][level:acceptance] Should contain configuration injected through openshift-node-performance profile [It]
    /go/src/github.com/openshift-kni/cnf-features-deploy/vendor/github.com/openshift-kni/performance-addon-operators/functests/1_performance/performance.go:287
    parameter kernel.hung_task_timeout_secs value is not 600.
      
    Expected
        <string>: 
    to equal
        <string>: 600
    /go/src/github.com/openshift-kni/cnf-features-deploy/vendor/github.com/openshift-kni/performance-addon-operators/functests/1_performance/performance.go:1193
```
An empty string returned from the remote exec command again.
There is an open bug on that.
I Will give it last try, if it won't work will try to evaluate other alternatives.

File: cnf-tests/pod-utils/cyclictest-runner/main.go (hunk 1 above)
Comment (yuvalk) [reply cnf-tests/pod-utils/cyclictest-runner/main.go:32]: this is great!

**Comment by Tal-or** [comment]
/retest
Looks like OLM failed to install some PTP and SR-IOV operators

**Pull request merged**